lib/watir/window_collection.rb

```diff
@@ -0,0 +1,98 @@
+module Watir
+  class WindowCollection
+    include Enumerable
+    include Waitable
+
+    def initialize(browser, selector = {})
+      unless selector.keys.all? { |k| %i[title url].include? k }
```

Comment:
Would it be hard to support `:element` in WindowCollection? It seems useful for Window, so would be nice here too.

Reply:
Oh not at all, this was just an oversight; and an indication that I don't have the right tests, either. :/
I squashed in some tests for `ElementCollection#new` and added element to this list.

Reply:
and... adding that test showed that my implementation was broken, so... yay tests.